### DISCUSSION TITLE
Server timings greyed out

### DISCUSSION THREAD

**Discussion by punekarmh12** [DaxStudio/DaxStudio#1112] [Q&A] [answered]
I am connected to a dataset published in service via the XMLA end point. It's read only XMLA. Server timings button is greyed out with a message " This command is currently greyed out - You must have Admin rights on the server to enable traces". I am the author of the dataset and have build permissions, admin role in the workspace. Any idea why the traces aren't enabled?

Thank you.

**Comment by dgosbell** [comment, marked as answer]
It will be because the XMLA endpoint is readonly. This will block CREATE commands and to start any of the traces DAX Studio needs to issue a create trace command and then subscribe to events from that trace object.
